subscriptions/fixed-price/checkout/node/src/server.ts

```diff
@@ -27,55 +33,114 @@ const endpointSecret = process.env.DIAGONAL_WEBHOOK_ENDPOINT_SECRET as string
 
 const diagonal = new Diagonal(apiKey)
 
-// Create an account for your customer
-app.post('/create-account/', async (req, res) => {
-  const email = req.body.email
-  // const name = req.body.name
-  // ...
+/**
+ * Plans
+ * Since you will be integrating fixed price subscriptions
+ * you will need some notion of plans/products, either defined locally, in your database,
+ * or using an external provider. In this example we choose to define plans locally.
```

Comment:
Nit: In this example, we choose to define plans statically in-memory

Reply:
We keep the plans in memory for quickly access clarity. Using inline database comments is ugly, and would like to avoid when possible.

Additionally keeping plans in memory will work decently well for simple fixed price applications .